x/feegrant/keeper/keeper.go

```diff
@@ -173,26 +156,34 @@ func (k Keeper) UseGrantedFees(ctx sdk.Context, granter, grantee sdk.AccAddress,
 	}
 
 	remove, err := grant.Accept(ctx, fee, msgs)
-	if err == nil {
-		ctx.EventManager().EmitEvent(
-			sdk.NewEvent(
-				types.EventTypeUseFeeGrant,
-				sdk.NewAttribute(types.AttributeKeyGranter, granter.String()),
-				sdk.NewAttribute(types.AttributeKeyGrantee, grantee.String()),
-			),
-		)
-	}
 
 	if remove {
-		k.RevokeFeeAllowance(ctx, granter, grantee)
-		// note this returns nil if err == nil
-		return sdkerrors.Wrap(err, "removed grant")
+		k.revokeFeeAllowance(ctx, granter, grantee)
```

Comment:
maybe add some notes explaining why we aren't getting the returned error from `k.revokeFeeAllowance`